src/common/constants.ts

```diff
@@ -13,4 +13,24 @@ export const SERVICES: Record<string, symbol> = {
   TRACER: Symbol('Tracer'),
   METER: Symbol('Meter'),
 };
+export enum Status {
```

Comment:
what's the difference from "enum OperationStatus" that is defined in  src/common/dataModels/enums.ts?

Reply:
Operation status include more statuses, it was implemented on old repo as is

export enum OperationStatus {
  PENDING = 'Pending',
  IN_PROGRESS = 'In-Progress',
  COMPLETED = 'Completed',
  FAILED = 'Failed',
  EXPIRED = 'Expired',
  ABORTED = 'Aborted',
}


export enum Status {
  PENDING = 'Pending',
  PROGRESS = 'In-Progress',
  COMPLETED = 'Completed',
  FAILED = 'Failed',